### ISSUE TITLE
go-libsql returns null für subsequent fields when selecting a datetime value

### ISSUE THREAD

**Issue by rfermann** [tursodatabase/go-libsql#18]
go-libsql fails fetching other columns in combination of a datetime field like this:
`SELECT id, datetime(start_date), datetime(end_date), active FROM table_name WHERE id = 1`

Every field specified after the first datetime field in the `SELECT`clause will be returned as null. If all datetime fields are removed from the `SELECT` clause, all other fields are returned with their actual values.
Every field specified before the first datetime field in the `SELECT`clause will be returned with its actual value.

Reproduction:

- create a new application, use official documentation as a starting point: https://docs.turso.tech/sdk/go/quickstart
- add the following code block after `defer connector.Close()` and run the program:
```	_, err = db.Exec("DROP TABLE IF EXISTS date_issue;")
	if err != nil {
		fmt.Println("Error dropping table:", err)
		os.Exit(1)
	}

	_, err = db.Exec(`
		CREATE TABLE IF NOT EXISTS date_issue (
		id         INTEGER PRIMARY KEY AUTOINCREMENT,
		start_date TEXT,
		end_date   TEXT,
		active	   bool    NOT NULL DEFAULT 0
		);`,
	)
	if err != nil {
		fmt.Println("Error creating table:", err)
		os.Exit(1)
	}

	var id int
	err = db.QueryRow(`
		INSERT INTO date_issue (
		    start_date, 
		    end_date, 
		    active
		) 
		VALUES (
		    datetime('now'), 
		    datetime('now'), 
		    false
		) RETURNING id`).Scan(&id)
	if err != nil {
		fmt.Println("Error creating table entry:", err)
		os.Exit(1)
	}

        type Date_Issue struct {
	        Id        int
	        StartDate time.Time
	        EndDate   time.Time
	        Active    bool
        }

	var dateIssue Date_Issue

	err = db.QueryRow(`
		SELECT id, datetime(start_date), datetime(end_date), active 
		FROM date_issue
		WHERE id = ?`,
		id).Scan(
		&dateIssue.Id,
		&dateIssue.StartDate,
		&dateIssue.EndDate,
		&dateIssue.Active,
	)
	if err != nil {
		fmt.Println("Error fetching data:", err)
		os.Exit(1)
	}

	fmt.Println(dateIssue)
```

The program errors with ```sql: Scan error on column index 1, name "datetime(end_date)": unsupported Scan, storing driver.Value type <nil> into type *time.Time```. The name of the affected column is based on the order in which the colums are specified in the `SELECT` clause: whatever column is specified right after the first date field will be reported here.

Removing the `datetime` function from the `SELECT` clause doesn't change this behaviour.

The remote-only client https://github.com/tursodatabase/libsql-client-go/libsql works fine with the given code example.

**Comment by lahickey** [comment]
I think https://github.com/tursodatabase/go-libsql/pull/20 has fixed this :+1:

**Comment by rfermann** [comment]
@lahickey you are right, issue has been fixed by #20. Thanks for digging into the problem and fixing it 👍
